server/app.js

```diff
@@ -12,4 +15,16 @@ app.use((err, req, res, next) => {
   }
 });
 
+app.get('/api', (req, res) => {
+  res.status(200).send(endpoints);
```

Comment:
This works, but it would be better if we have a controller to send the response and endpoints 👍